crates/weaver_forge/src/formats/markdown.rs

```diff
@@ -81,10 +104,19 @@ impl RenderContext {
     /// Add a blank line if the current markdown buffer
     /// does not end already with a double newline.
     fn add_cond_blank_line(&mut self) {
+        // TODO - This is a workaround for not truly
+        // refactoring word-wrap vs. regular add-text.
         if !self.markdown.ends_with("\n\n") && !self.markdown.is_empty() {
-            self.markdown.push('\n');
+            let _ = self
```

Comment:
Why not using the `add_blank_line` defined just after?

Reply:
It adds a prefix, this does not.  `add_blank_line` should be fine, I may need to go update the prefix handling code.